site/en/docs/crux/historical-api/index.md

```diff
@@ -0,0 +1,617 @@
+---
+# Required
+layout: 'layouts/doc-post.njk'
+
+# Required
+title: CrUX Historical API
+
+# Required
+# This appears in the ToC of the project landing page at
+# /docs/[project-name]/. It also appears in the <meta description> used in
+# Google Search.
+description: >
+  Learn how to construct requests to and parse responses from the CrUX Historical API.
+
+# Optional
+# This appears below the title and is an optional teaser
+subhead: >
+  The CrUX Historical API gives low-latency access to aggregated real-user experience data at page and origin granularity.
+
+# Required
+date: 2023-02-07
+
+# Optional
+# Include an updated date when you update your post
+# updated: 2023-02-07
+
+# Optional
+# How to add a new author
+# https://developer.chrome.com/docs/handbook/how-to/add-an-author/
+# authors:
+
+# Optional
+# How to a new tag
+# https://developer.chrome.com/docs/handbook/how-to/add-a-tag/
+tags:
+  - web-vitals
+  - crux
+---
+
```

Comment:
suggestion: What do you think about adding a quick setup/start section here?

Something similar to how it is done in Page Speed Insights for users that want to try it out fast (https://developers.google.com/speed/docs/insights/v5/get-started)

Reply:
As I say, we have a blog post (that will be linked from here), that's a more gentle introduction. Working on adding that to this PR. This doc is supposed to me more of a reference doc.